otmonitor/rootfs/usr/share/otmonitor/html/alert.html.tmpl

```diff
@@ -0,0 +1,67 @@
+%# -*- html -*-
+% global cfg theme
+<!DOCTYPE html>
+<html lang="en">
+<head>
+[include header.inc]
+</head>
+<body>
+[include navbar.inc]
+
+% if {![info exists post]} {set post 0}
+% if {[dict exists $post alert,pressurehigh ""]} {set cfg(alert,pressurehigh) [dict get $post alert,pressurehigh {}]}
+% if {[dict exists $post alert,pressurelow ""]} {set cfg(alert,pressurelow) [dict get $post alert,pressurelow {}]}
+% if {[dict exists $post alert,roomcold ""]} {set cfg(alert,roomcold) [dict get $post alert,roomcold {}]}
+
+<div class="container">
+	<div class="row-content">
+<h1>Alert</h1>
+<div> 
+
+<form class="form-horizontal" role="form" enctype="multipart/form-data" method="post" action="alert.html">
+<div class="form-group">
+    <label class="control-label col-sm-2" for="alert,pressurehigh">Alert pressurehigh:</label>
+    <div class="col-sm-4">
+		<input type="text" class="form-control" name="alert,pressurehigh" value="$cfg(alert,pressurehigh)">
```

Comment:
Would you mind to run these html templates through some "prettify" tool. Like https://www.freeformatter.com/html-formatter.html#ad-output would be good enough or just using VS Code. If too much work, I can do it ;)

Reply:
Absolutely :) I'm already doing some prettify as we type 👍